src/config.js

```diff
@@ -1,8 +1,4 @@
 import convict from "convict";
-import path from "path";
-import * as url from "url";
-const __filename = url.fileURLToPath(import.meta.url);
-const __dirname = url.fileURLToPath(new URL(".", import.meta.url));
 
 export default convict({
   privateKeyFileName: {
```

Comment:
should there be a comment here to remove the param if not used, or will the null value just be ignored? like if privateKeyFileName was not set in the .env